pkg/master/thirdparty_controller_test.go

```diff
@@ -110,15 +136,15 @@ func TestSyncAPIs(t *testing.T) {
 				},
 			},
 			apis: []string{
-				"/apis/example.com",
-				"/apis/example.com/v1",
-				"/apis/example.co",
-				"/apis/example.co/v1",
+				"/apis/example.com/foos",
+				"/apis/example.com/v1/foos",
+				"/apis/example.co/foo",
```

Comment:
Why remove the strings `apis/example.com` and `apis/exmample.com/v1`? Those should be part of expectedRemoved as well?